src/app/_components/Icons/Facebook.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import p from 'prop-types';
+
+const FacebookComponent = ({ className }) => (
```

Comment:
🟢 can't we use something like react-icons for this?

Reply:
I'd like to, but react-icons don't have all icons we need.